IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeoScienceObservation/Documentation.md

```diff
@@ -0,0 +1 @@
+Detailed collected information, including measured parameters, descriptions etc related to geoscientific observations. that can be assigned to physical or spatial elements using _IfcRelAssignsToProduct_.
```

Comment:
```suggestion
Detailed collected information, including measured parameters, descriptions etc. related to geoscientific observations. that can be assigned to physical or spatial elements using _IfcRelAssignsToProduct_.
```

Reply:
Should not be a period after "observations" since sentence continues.